src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -174,9 +348,14 @@ export default class FileField extends React.Component {
                   {!file.uploading && <p>{uiOptions.itemDescription}</p>}
                   {!file.uploading && (
                     <span>
-                      <strong>{file.name}</strong>
+                      <strong>{file.name}</strong>{' '}
+                      {file.unlocked && '(unlocked)'}
```

Comment:
We may need approval from a content person on this. I can imagine users being confused when they see `(unlocked)` next to something they uploaded and worrying about that.